_articles/faq.md

```diff
@@ -1,8 +1,9 @@
 ---
-title: "FAQ"
+title: FAQ
 layout: article
 excerpt: What is Synapse?
 order: 4
+published: true
```

Comment:
@allaway did prose add this to the yaml? I don't think this will be interpreted in the build.